cmd/flow-aggregator/options.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"strings"
+	"time"
+
+	"github.com/spf13/pflag"
+	"gopkg.in/yaml.v2"
+)
+
+type Options struct {
+	// The path of configuration file.
+	configFile string
+	// The configuration object
+	config *FlowAggregatorConfig
+	// IPFIX flow collector address
+	flowCollectorAddr net.Addr
+	// Flow export interval of the flow aggregator
+	exportInterval time.Duration
+}
+
+func newOptions() *Options {
+	return &Options{
+		config: new(FlowAggregatorConfig),
+	}
+}
+
+// addFlags adds flags to fs and binds them to options.
+func (o *Options) addFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&o.configFile, "config", o.configFile, "The path to the configuration file")
+}
+
+// complete completes all the required options.
+func (o *Options) complete(args []string) error {
+	if len(o.configFile) > 0 {
+		c, err := o.loadConfigFromFile(o.configFile)
+		if err != nil {
+			return err
+		}
+		o.config = c
+	}
+	return nil
+}
+
+// validate validates all the required options.
+func (o *Options) validate(args []string) error {
+	if len(args) != 0 {
+		return errors.New("no positional arguments are supported")
+	}
+	if o.config.FlowCollectorAddr == "" {
+		return fmt.Errorf("IPFIX flow collector address should be provided")
+	} else {
```

Comment:
if the address parsing logic can be unified with the agent, please consider doing it in a subsequent PR